src/abstractdataframe/show.jl

```diff
@@ -24,12 +24,24 @@ end
 Render a value to an IO object. Unlike
 `show`, render strings without surrounding quote marks.
 """
-ourshow(io::IO, x::Any) =
-    show(IOContext(io, :compact=>get(io, :compact, true), :typeinfo=>typeof(x)), x)
+function ourshow(io::IO, x::Any)
+    ctxt = IOContext(io, :compact=>get(io, :compact, true), :typeinfo=>typeof(x))
+    show(ctxt, "text/plain", x)
+end
+
 ourshow(io::IO, x::AbstractString) = escape_string(io, x, "")
 ourshow(io::IO, x::Symbol) = ourshow(io, string(x))
 ourshow(io::IO, x::Nothing) = nothing
 
+# https://github.com/JuliaLang/julia/pull/34730
+# https://github.com/JuliaLang/julia/pull/34741
+if VERSION < v"1.5.0-DEV.261" || VERSION < v"1.5.0-DEV.266"
```

Comment:
just to understand - why this is not the same as simply `if VERSION < v"1.5.0-DEV.266"`?

Reply:
I just did it this way to record the changes to `AbstractChar` and `Irrational` occurred in different revisions. I can add a comment to make that more explicit or just drop that if it it seems unnecessary.

I may need to split this up further due to how `Irrational` works in Julia 1.0. Will post details

Reply:
I've added the comment. I can drop the `||` if you prefer still but I figured it doesn't really matter as this check only happens at compile time.

Reply:
Thank you. I have approved the PR. I am just waiting for @nalimilan to have a look at it and comment or merge.